Asset/Asset.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/*
+ * This file is part of the Integrated package.
+ *
+ * (c) e-Active B.V. <integrated@e-active.nl>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Integrated\Bundle\AssetBundle\Asset;
+
+/**
+ * @author Ger Jan van den Bosch <gerjan@e-active.nl>
+ */
+class Asset
+{
+    /**
+     * @var string
+     */
+    protected $content;
+
+    /**
+     * @var bool
+     */
+    protected $inline = false;
+
+    /**
+     * @param string $content
+     * @param bool $inline
+     */
+    public function __construct($content, $inline = false)
+    {
+        $this->setContent($content);
+        $this->setInline($inline);
```

Comment:
Is there a reason to use the setters instead of directly writing in the properties? A constructor should initialize the properties.